src/applications/mhv-secure-messaging/components/ComposeForm/EditPreferences.jsx

```diff
@@ -28,7 +28,7 @@ const EditPreferences = () => {
         onClick={() => setEditListModal(true)}
         type="button"
       >
-        <div className="vads-u-display--flex vads-u-flex-direction-row vads-u-justify-content--center">
+        <div className="vads-u-display--flex vads-u-flex-direction-row vads-u-justify-content--center vads-u-color--white">
```

Comment:
This fixes an issue where the button text was blue on a blue button background. The color utilities were migrated to CSS Library in an [earlier PR](https://github.com/department-of-veterans-affairs/vets-website/pull/31189) and I think that caused a specificity issue with the white text `color` not applying anymore.

Reply:
@jamigibbs I am noticing in `staging` that this introducing an issue where it's a white font color on white background 
![image](https://github.com/user-attachments/assets/a7e52072-acb2-4a70-a238-81b7bf8ff5be)

Reply:
Interesting. Originally and as designed, the button should have balck border and font color, with a white background. I am wondering if there are other formation styling changes that may have impacted styling for this button to become blue as you described. No worries, I have already pushed a fix for that. Jsut wanted to flag in case this may be an issue somewhere else